neo/test/iotest/test_cedio.py

```diff
@@ -0,0 +1,18 @@
+import unittest
+
+from neo.io import CedIO
+from neo.test.iotest.common_io_test import BaseTestIO
+
+
+class TestCedIO(BaseTestIO, unittest.TestCase, ):
+    ioclass = CedIO
+    entities_to_test = [
+        'spike2/m365_1sec.smrx'
+    ]
+    entities_to_download = [
+        'spike2/m365_1sec.smrx'
```

Comment:
Shouldn't this rather be rather imported from the rawio tests than hard coded twice?
```suggestion
    entities_to_download = [
        'spike2'
```